src/stories/lightButton.css

```diff
@@ -0,0 +1,20 @@
+.light-button {
```

Comment:
Need to delete this additional file. it is copy of /06_shared/ui/button/lightButton/lightButton.scss